### ISSUE TITLE
Add syntax sugar

### ISSUE THREAD

**Issue by ai** [logux/vuex#9]
Can we have syntax sugar to reduce code

```js
	mouted () {
		// Load current counter from server and subscribe to counter changes
		this.$store.commit.sync({
			type: 'logux/subscribe',
			channel: 'counter'
		})
	},
	beforeDestroy () {
		// Unsubscribe from counter before component destroyed
		this.$store.commit.sync({
			type: 'logux/unsubscribe',
			channel: 'counter'
		})
	},
```

You can use [`useSubscription()`](https://logux.io/guide/concepts/subscription/#usesubscription) as an example. Syntax sugar should accept channel names `'user/15', 'user/14'`, objects `{ channel: 'user/15', fields: ['name'] }` and generated dynamically (`props.ids.map(i => 'user/' + i)`).

**Comment by euaaaio** [comment]
I am already thinking about it.

**Comment by euaaaio** [comment]
Released in v0.2.0
